### ISSUE TITLE
ENH: Include option to specify landing page / 404 custom html assets for html builder

### ISSUE THREAD

**Issue by mmcky** [QuantEcon/sphinxcontrib-jupyter#231]
It would be nice to have the ability to specify path to `html` static assets you may want to use when constructing the site, such as a more sophisticated frontpage (`index.html`) and a `404.html` etc.

```
jupyter_html_index = "path"
jupyter_html_404 = "path"
```

**Comment by mmcky** [comment]
this is currently supported

If you place files in ``theme/html`` then they will be coped over to the website build directory ``_build/website/jupyter_html``

**Comment by mmcky** [comment]
Will implement the following options

```
jupyter_theme = 'name'
jupyter_theme_path = <path>

jupyter_template_path = <path>
jupyter_template_html = <filename>
jupyter_template_pdf = <filename
```

this will support custom locations for `theme` or `templates`

**Comment by mmcky** [comment]
Closed with #233
